doc/chromebooks/luks-installation.md

```diff
@@ -0,0 +1,433 @@
+# Step -1
+
+- **before proceeding** with installation it is recommended to first [set gbb flags](./setting_gbb_flags.md) (especially if you install it directly onto chromebook memory)
+
+- if you want **diffrent username** (default is linux)
+
+login as linux (password "changeme"), open terminal and type
+
+```
+sudo passwd root
+```
+this will let you change root password
+
+logout and login as root and run script
+```
+/scripts/rename-default-user.sh mypreferredusername
+```
+
+_Note. if the user is used by some process just kill it (```kill <process id>```) and run the command again which should throw a group error but don't worry about it :3_
+
+# Let's just do it
+
+_Note. this installetion doesn't have to be performed on internal memory, it can also be done on another usb or sd card_
+
+_Important. it is asummed you do all this as root use ```su root``` or ```sudo -i```_
+
+1. start by listing all available disks
+
+```
+lsblk
+```
+the output should look like this
+```
+NAME         MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
+mtdblock0     31:0    0     8M  0 disk 
+mmcblk1      179:0    0 116,5G  0 disk <- this is a disk
+├─mmcblk1p1  179:1    0    64M  0 part <- this is a partition
+├─mmcblk1p2  179:2    0    64M  0 part 
+├─mmcblk1p3  179:3    0   512M  0 part /boot
+├─mmcblk1p4  179:4    0 107,9G  0 part /
+└─mmcblk1p5  179:5    0     8G  0 part [SWAP]
+mmcblk1boot0 179:32   0     4M  1 disk 
+mmcblk1boot1 179:64   0     4M  1 disk
+```
+
+now in order to make it simple to just copy and paste the commands
+we will export the correct drive as a environment variable but i still encourage you to read as much as possible here
+
+_Note. remmber to do it whenever you want to repeat any of steps below_
+
+- if you want to install onto ```mmcblk0``` or ```mmcblk1``` (number is diffent on diffrent chromebooks)
+```
+export disk=mmcblk0
+export part=mmcblk0p
+```
+or
+```
+export disk=mmcblk1
+export part=mmcblk1p
+```
+respecitively
+
+- if you want to install on sda or similar
+```
+export disk=sda
+export part=sda
+```
+
+3. prepare the disk
+
+wipe the target disk first
+```
+apt-get install cgpt # should be already installed
+sgdisk -Z /dev/${disk}
+partprobe /dev/${disk}
+sgdisk -C -e -G /dev/${disk}
+partprobe /dev/${disk}
+cgpt create /dev/${disk}
+partprobe /dev/${disk}
+```
+
+create kernel partition
+```
+cgpt add -i 1 -t kernel -b 8192 -s 65536 -l KernelA -S 1 -T 2 -P 10 /dev/${disk}
+cgpt add -i 2 -t kernel -b 73728 -s 65536 -l KernelB -S 0 -T 2 -P 5 /dev/${disk}
+```
+
```

Comment:
we should maybe bring over the special commands required for veyron from https://github.com/hexdump0815/imagebuilder/blob/main/doc/install-to-emmc-with-luks-full-disk-encryption.txt#L48-L57 here as well so that this chromebook is also covered properly

Reply:
wasn't sure if veyron shouldn't be treated as legacy stuff so i left it out but can add